calculator_program.c

```diff
@@ -0,0 +1,143 @@
+#include<stdio.h>
+#include <stdbool.h> 
+#include<stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+bool checkValid(char *str){
+    int i=0;
+    while(str[i] != '\0'){
+        if(str[i]!='+' && str[i]!='-' && str[i]!='*' && str[i]!='/' && str[i]!=' ' && !isdigit(str[i])){
+            return false;
+        }
+        i++;
+    }
+
+    return true;
+}
+
+int priority(char ch){
```

Comment:
Focus on Variable and function names.
Name in a way that its functionality can be understood just by reading name.